src/common/geo/io/wkb/WKBReader.cpp

```diff
@@ -0,0 +1,156 @@
+/* Copyright (c) 2018 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "common/geo/io/wkb/WKBReader.h"
+
+namespace nebula {
+
+StatusOr<std::unique_ptr<Geometry>> WKBReader::read(std::string wkb) const {
```

Comment:
I check the usage, `read` could just return `Geometry` instead of `std::unique_ptr<Geometry>`.
Just a todo.

Reply:
Seems return `Geometry` is also ok.

Reply:
I have change it to return `Geometry`.